modules/ROOT/pages/activity-tracking.adoc

```diff
@@ -1,87 +1,129 @@
-. Locate the message flow whose activity you want to track.
-+
-Use the search box to find the transmission based on criteria such as:
-+
-* Trading partner name
-* Message type
-* File name
-* Control number
-* Endpoint
-* Received date of the file +
-. You can then use the filters to find transmissions based on:
-* *Type* +
-Type of message transmission:
-** *Both directions*
-** *Sent to partners*
-** *Received from partners*
-* *From Date* +
-Filter for transmissions sent or received after this date.
-* *To Date* +
-Filter for transmissions sent or received up to and including this date.
-* *Ack Status* +
-Filter for transmissions based on their acknowledgment status:
-** *All*
-** *Overdue*
-** *Rejected* +
-* *Message Processing Status* +
-Filter for transmissions based on their message processing status:
-** *All*
-** *Delivered*
-** *Failed*
-** *Processing*
-** *Received*
-* *Partner name* +
-Filter for transmissions based on your partner's name. Only five partners appear in the *Partner name* list; however, you can enter characters in the search box to access partners that do not appear in the list.
-* *Partner message type* +
-Filter for transmissions based on the message type, such as `X12 - 850`. +
-* *Host message type* +
-Filter for transmissions based on the host message type, such as `JSON host-purchase-order`.
-. Select the transmission that matches your search criteria and review the summary information:
-+
-* The success or failure status of the transmission (indicated by either a checkmark for success or a circle with an exclamation point for failure)
-* The name of the received file
-* The control number associated with the file (if available)
-* The receiving endpoint
-* The received date and time of the file
-* The types of messages received (depending on the configured message flows)
-* The number of messages received in the file
-* Transmission flow +
-The *Transmission Flow* section of the summary information shows:
-* Success or failure of the transmission
-* Acknowledgment status +
-If the configured outbound message flow expects an acknowledgment from the trading partner or if the configured inbound message flow returns an acknowledgment to the trading partner, then you can view the acknowledgment (997 and TA1) payloads in the transmission detail.
+. Search for the transmission.
 +
-If you have not received an acknowledgment from your trading partner or if your trading partner sent a rejection acknowledgment, then you can review the acknowledgment status of outbound transactions.
-* Transformation and routing information
-+
-An exclamation point (!) appearing next to any of the rectangle boxes in the *Transmission Flow* section indicates an issue with the transmission.
-. Click *View Payload* to view and download the originally received payload.
+When you search for the transission, you can filter the results based on the following criteria:
+
+* Message type, such as `my-po-ack:ORD-9892330`
```

Comment:
The first filter with the label "Type", is actually direction - with dropdown menu options "Receive from partners" and "Send to partners".